tmt/steps/prepare/install.py

```diff
@@ -165,6 +165,15 @@ def install_debuginfo(self) -> None:
 
     def install(self) -> None:
         """ Perform the actual installation """
+        try:
+            self._install()
+        except Exception:
+            # Refresh cache in case of recent but not updated change do repodata
+            self.guest.package_manager.refresh_metadata()
+            self._install()
```

Comment:
I wonder whether should capture the first exception as well - this way, the original one is lost, with all its data. How about something like this?

```python
try:
    self._install()

except Exception as exc1:
    # We do not have any special handling for exceptions raised by the following code.
    # Wrapping them with try/except gives us a chance to attach the original exception
    # to whatever the code may raise, and therefore preserve the information attached
    # to the original exception.
    try:
        # Refresh cache in case of recent but not updated change do repodata
        self.guest.package_manager.refresh_metadata()
        self._install()

    except Exception as exc2:
        raise exc2 from ex1
```

Reply:
You're right, that seems like an unnecessary loss of data - better to catch them both. I used your code, if that's alright:)